docs/reference/commandline/plugin_inspect.md

```diff
@@ -21,7 +21,7 @@ in a JSON array.
 
 ### Inspect a plugin
 
-The following example example inspects the `tiborvass/sample-volume-plugin` plugin:
+The following example inspects the `tiborvass/sample-volume-plugin` plugin:
```

Comment:
Hm... we need to enable the "dupwords" linter. But perhaps it only checks Go code, and not markdown files 🤔